app/components/list-uploaded-assets/ListUploadedAssets.tsx

```
@@ -0,0 +1,89 @@
+  }
+
+  return (
+    <div className="mx-0 flex flex-col">
+      {isLoading && filteredCreatorAssets.length === 0 && (
+        <p className="my-6 text-[--color-brand-red-shade]">Loading assets...</p>
+      )}
+
+      {filteredCreatorAssets.length > 0 && (
+        <table
+          className="my-6 w-full table-auto"
+          role="grid"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<table>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)